File: backend/app/models/Reserva.php
<?php  defined('INITIALIZED') OR exit('You cannot access this file directly');

class Reserva extends Model {
	
	private $id;
	private $dataUso;
	private $dataHoraReserva;
	private $ativo;
	private $sala;
	private $professor;
	private $horario;

    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getDataUso()
    {
        return $this->dataUso;
    }

    public function setDataUso($dataUso)
    {
        $this->dataUso = $dataUso;
    }

    public function getDataHoraReserva()
    {
        return $this->dataHoraReserva;
    }

    public function setDataHoraReserva($dataHoraReserva)
    {
        $this->dataHoraReserva = $dataHoraReserva;
    }

    public function getAtivo()
    {
        return $this->ativo;
    }

    public function setAtivo($ativo)
    {
        $this->ativo = $ativo;
    }

    public function getSala()
    {
        return $this->sala;
    }

    public function setSala($sala)
    {
        $this->sala = $sala;
    }

    public function getProfessor()
    {
        return $this->professor;
    }

    public function setProfessor($professor)
    {
        $this->professor = $professor;
    }

    public function getHorario()
    {
        return $this->horario;
    }

    public function setHorario($horario)
    {
        $this->horario = $horario;
    }

	
}

File: Lixo/frontend/src/models/Reserva.java
<?php

defined('INITIALIZED') OR exit('You cannot access this file directly');

class Reserva extends Model {

    private $id;
    private $dataUso;
    private $dataHoraReserva;
    private $ativo;
    private $idSala;
    private $idProfessor;
    private $horario;

    function getId() {
        return $this->id;
    }

    function getDataUso() {
        return $this->dataUso;
    }

    function getDataHoraReserva() {
        return $this->dataHoraReserva;
    }

    function getAtivo() {
        return $this->ativo;
    }

    function getIdSala() {
        return $this->idSala;
    }

    function getIdProfessor() {
        return $this->idProfessor;
    }

    function getHorario() {
        return $this->horario;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setDataUso($dataUso) {
        $this->dataUso = $dataUso;
    }

    function setDataHoraReserva($dataHoraReserva) {
        $this->dataHoraReserva = $dataHoraReserva;
    }

    function setAtivo($ativo) {
        $this->ativo = $ativo;
    }

    function setIdSala($idSala) {
        $this->idSala = $idSala;
    }

    function setIdProfessor($idProfessor) {
        $this->idProfessor = $idProfessor;
    }

    function setHorario($horario) {
        $this->horario = $horario;
    }

}
